doc/architecture/04solution.md

```diff
@@ -86,3 +121,21 @@ While the lock is active, other users see these layers read-only.
 Upon coming back online the changes made are synchronized and the lock is released.
 
 See also [offline use case](../usecases/assigned/offline.md).
+
+## Identity and Access Management
+
+Only the landing page can be seen without registration.
+We use [Keycloak](https://www.keycloak.org/) for Identity and Access Management with 3 roles:
+
+1. default-roles-permaplant (only [public maps and the membership application](../usecases/current/membership_application.md) can be visited)
+2. member (can do everything but administration tasks)
+3. admin (possibility to change data from other users, as needed for onboarding or offboarding)
+
+## Privacy
+
+In general all data must to stay within the organisation (Verein) and can only be seen by members or even only admins.
```

Comment:
```suggestion
In general all data must stay within the organisation (Verein) and can only be accessed by members or admins.
```
This sentence makes it sound like every member of the orgranization can access everyone elses data without restriction.
I think you meant to say that members can only access their own data or data that was explicitly shared with them, while admins can access everything.

Reply:
```diff
-In general all data must stay within the organisation (Verein) and can only be accessed by members or admins.
-Personally identifiable information should be stored in Keycloak.
+In general all data must stay within the organisation (Verein).
+Members see other members' data only if they allowed it.
+Data which makes a person identifiable should be in Keycloak.
```

254d7bcc933ccd7d60148962041c8af6c3e721ee